www/include/reporting/dashboard/DB-Func.php

```diff
@@ -244,6 +244,28 @@ function getLogInDbForHostSVC($host_id, $start_date, $end_date, $reportTimePerio
      */
     $services_ids = $centreon->user->access->getHostServiceAclConf($host_id, 'broker');
     $svcStr = "";
+    $status = ["OK", "WARNING", "CRITICAL", "UNKNOWN", "UNDETERMINED", "MAINTENANCE"];
+    /*
+    * $hostServiceStats["average"] will contain services stats average
+    */
+    foreach ($status as $name) {
+        switch ($name) {
+            case "UNDETERMINED":
+                $hostServiceStats["average"]["UNDETERMINED_TP"] = 0;
+                break;
+            case "MAINTENANCE":
+                $hostServiceStats["average"]["MAINTENANCE_TP"] = 0;
+                break;
+            default:
+                $hostServiceStats["average"][$name . "_MP"] = 0;
+                $hostServiceStats["average"][$name . "_TP"] = 0;
+                $hostServiceStats["average"][$name . "_A"] = 0;
+                break;
+        }
```

Comment:
Is this supposed to be equivalent to former lines 265 to 274 ?
If yes, 'UNDERTERMINED_MP' and 'MAINTENANCE_MP' definitions are missing